tidb-cloud/manage-user-access.md

```diff
@@ -149,73 +165,50 @@ To invite a member to a project, perform the following steps:
 >
 > The verification link in the email will expire in 24 hours. If your user doesn't receive the email, click **Resend**.
 
-## Configure member roles
+## Remove a project member
+
+If you are the owner of an organization, you can remove project members. Otherwise, skip this section.To remove a member from a project, perform the following steps:
```

Comment:
```suggestion
If you are the owner of an organization, you can remove project members. Otherwise, skip this section.

To remove a member from a project, perform the following steps:
```